src/CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ/MassTransitRabbitMqExtensions.cs

```diff
@@ -0,0 +1,76 @@
+﻿using CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ;
+using MassTransit;
+using MassTransit.Monitoring;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System.Reflection;
+
+namespace Microsoft.Extensions.Hosting;
+
+/// <summary>
+/// MassTransitClientExtensions provides extension methods for configuring MassTransit in a client application.
+/// </summary>
+public static class MassTransitRabbitMqExtensions
+{
+    /// <summary>
+    /// Configures MassTransit with RabbitMQ integration for the client side,
+    /// using the same configuration used by the hosting environment.
+    /// </summary>
+    /// <param name="builder">The client IHostApplicationBuilder.</param>
+    /// <param name="name">A unique name for the RabbitMQ instance.</param>
+    /// <param name="configure">Optional action to override default settings.</param>
+    public static void AddMassTransitRabbitMq(
+        this IHostApplicationBuilder builder,
+        string name,
+        Action<MassTransitRabbitMqOptions>? configure = null)
+    {
+        ArgumentException.ThrowIfNullOrWhiteSpace(name, nameof(name));
+
+        // Load options from configuration
+        var serviceProvider = builder.Services.BuildServiceProvider();
```

Comment:
The serviceprovider is an IDisposable. Shouldn't we dispose this?

Reply:
We shouldn't be resolving a service provider within this phase. To access configuration, use `builder.Configuration`: https://github.com/CommunityToolkit/Aspire/blob/main/src/CommunityToolkit.Aspire.OllamaSharp/AspireOllamaSharpExtensions.cs#L106-L109